src/views/Home.vue

```diff
@@ -66,6 +80,9 @@
     mounted () {
       // console.log('currentUser', currentUser())
       // console.log('isLoggedIn', isLoggedIn())
+      if (this.$store.state.isLoggedIn) {
+        fetchFollowing(this.$store, { subject: _.get(this.$store.state.profile, [ 'id' ]), resource: 'post' })
```

Comment:
What will this function return?

Reply:
Dispatch a action that will return a Promise object, which will getting a following posts list from user id.